users/details/script.js

```diff
@@ -108,6 +108,43 @@ function generateUserData(userData) {
     type: 'div',
     classList: ['user-details-main'],
   });
+  const socials = generateSocialMediaLinksList();
+
+  if (userData?.profileURL) {
```

Comment:
what if userData.profileURL property is undefined

Reply:
this button is not created for every user only for those whose intro page exist that's why this condition exist
ankush requested I want option of the button for users whose intro page exist 
that's why if its undefined it means intro page for that user don't exist and button will not be created for that user